.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,49 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: 'CodeQL'
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    paths:
+      - src
+    paths-ignore:
+      - src/node_modules
+      - '**/*.test.ts'
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: ['javascript']
```

Comment:
We do not have any javascript in otomi-core

Reply:
Not yet